README.md

```diff
@@ -0,0 +1,7 @@
+# BC Parks Foundation
+
```

Comment:
- Add tech used
- Add project description
- Add screenshots
- Add feature considerations
- Add notes for the user
- Add installation steps